### PR TITLE
#372 sort pinned subreddits

### PR DESCRIPTION
**Pull request by chippen**
This is almost done. I had two issues.
1. I did not understand how to make it refresh when the setting was changed. Now you have to do a manual refresh. (I solved the refresh part in a1e36d9)
2. I don't understand why you don't have to use android:summary="%s" (I had to)

### CODE REVIEW

File: src/main/java/org/quantumbadger/redreader/common/PrefsUtility.java [hunk 1]
```diff
@@ -404,6 +405,14 @@ public static PostSort pref_behaviour_postsort(final Context context, final Shar
 		return PostCommentListingURL.Sort.valueOf(General.asciiUppercase(getString(R.string.pref_behaviour_commentsort_key, "best", context, sharedPreferences)));
 	}
 
+	public enum PinnedSubredditSort {
+		NAME, DATE
+	}
+
+	public static PinnedSubredditSort perf_behaviour_pinned_subredditsort(final Context context, final SharedPreferences sharedPreferences) {
```
Comment (QuantumBadger) [inline src/main/java/org/quantumbadger/redreader/common/PrefsUtility.java]: Minor typo (pref vs perf :P)

**Comment by QuantumBadger** [comment]
This looks great, thanks! There's a minor typo which I've commented on, but other than that this looks ready to merge!

I think setting the summary to "%s" sets the text underneath the preference to its value (although there's a RR-specific mechanism for that here: https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/settings/SettingsFragment.java#L71)

**Comment by chippen** [comment]
I fixed the typo and the summary

**Comment by QuantumBadger** [comment]
Perfect, thank you!

**Pull request merged**